pkg/csi_driver/multishare_controller.go

```diff
@@ -34,27 +34,30 @@ import (
 const (
 	modeMultishare = "modeMultishare"
 
-	methodCreateVolume         = "CreateVolume"
-	methodDeleteVolume         = "DeleteVolume"
-	methodExpandVolume         = "ExpandVolume"
-	ecfsDataPlaneVersionFormat = "GoogleReserved-CustomVMImage=clh.image.ems.path:projects/%s/global/images/ems-filestore-scaleout-%s"
+	methodCreateVolume              = "CreateVolume"
+	methodDeleteVolume              = "DeleteVolume"
+	methodExpandVolume              = "ExpandVolume"
+	ecfsDataPlaneVersionFormat      = "GoogleReserved-CustomVMImage=clh.image.ems.path:projects/%s/global/images/ems-filestore-scaleout-%s"
+	ecfsCustom100sharesConfigFormat = "GoogleReservedOverrides={\"CustomMultiShareConfig\":{\"MaxShareCount\": %d, \"MinShareSizeGB\":%d}}"
 )
 
 // MultishareController handles CSI calls for volumes which use Filestore multishare instances.
 type MultishareController struct {
-	driver          *GCFSDriver
-	fileService     file.Service
-	cloud           *cloud.Cloud
-	opsManager      *MultishareOpsManager
-	volumeLocks     *util.VolumeLocks
-	ecfsDescription string
-	isRegional      bool
-	clustername     string
+	driver                           *GCFSDriver
+	fileService                      file.Service
+	cloud                            *cloud.Cloud
+	opsManager                       *MultishareOpsManager
+	volumeLocks                      *util.VolumeLocks
+	ecfsDescription                  string
+	isRegional                       bool
+	clustername                      string
+	featureMaxSharePerInstance       bool
+	descOverrideMaxSharesPerInstance string
```

Comment:
what is the meaning of `descOverride` for the naming here?

Reply:
This is filestore instance description override, that can be used to enable features in filestore. I ll put a comment